UnitsNet/UnitsNet.csproj

```diff
@@ -60,6 +60,9 @@
     <Compile Remove="UnitDefinitions\**" />
     <EmbeddedResource Remove="UnitDefinitions\**" />
     <None Remove="UnitDefinitions\**" />
+    <None Include="..\Common\UnitDefinitions\TemperatureGradient.json">
+      <Link>Common\UnitDefinitions\TemperatureGradient.json</Link>
+    </None>
```

Comment:
This was probably accidental, but I can't push to your master branch so I will fix after merging